### PR TITLE
[IMP] POS: path to configure payment terminals

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
task-3684488
Still have to jump in in 16.0 for SIX

### CODE REVIEW

File: content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst [hunk 1]
```diff
@@ -19,23 +19,8 @@ your terminal following the steps described on your terminal's screen.
...
-   to appear.
-
-Finally, fill the mandatory fields with an :guilabel:`Adyen API key`, and an :guilabel:`Adyen
-Terminal Identifier`.
-
 Generate an Adyen API key
-~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------
 
 The **Adyen API key** is a key used to authenticate your requests. To generate an API key, go to
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst:25]:
```suggestion
The **Adyen API key** is used to authenticate your requests. To generate an API key, go to
```
It is redundant to write key twice

File: content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst [hunk 2]
```diff
@@ -73,12 +58,21 @@ For Odoo to know when a payment is made, you must set the terminal **Event URLs*
    `/pos_adyen/notification` in the :guilabel:`Event URLs` field; and
 #. click :guilabel:`Save` at the bottom of the screen to save the changes.
 
-Add a new payment method
-========================
+Configure the payment method
+----------------------------
+
+First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst:64]:
```suggestion
Start by :ref:`enabling the payment terminal <terminals/configuration>` and :doc:`creating the related
```
If you use "First", there must be a "Second" somewhere, here you only have "First" and "Finally"

File: content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst [hunk 3]
```diff
@@ -73,12 +58,21 @@ For Odoo to know when a payment is made, you must set the terminal **Event URLs*
...
+payment method <../../payment_methods>`. Set the journal as :guilabel:`Bank` and select
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst:65]:
```suggestion
payment method <../../payment_methods>`. Set the journal type as :guilabel:`Bank` and select
```

File: content/applications/sales/point_of_sale/payment_methods/terminals/ingenico.rst [hunk 4]
```diff
@@ -35,19 +35,16 @@ Box* (you can find it on the form view of your IoT Box). Then, enter
 Configure the payment method
 ----------------------------
 
-First, go in the general settings of the POS app, and activate the
-Ingenico setting.
+First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/ingenico.rst:38]:
```suggestion
First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```
Same as above

File: content/applications/sales/point_of_sale/payment_methods/terminals/ingenico.rst [hunk 5]
```diff
@@ -35,19 +35,16 @@ Box* (you can find it on the form view of your IoT Box). Then, enter
...
+payment method <../../payment_methods>`. Set the journal as :guilabel:`Bank` and select
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/ingenico.rst:39]:
```suggestion
payment method <../../payment_methods>`. Set the journal type as :guilabel:`Bank` and select
```

File: content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst [hunk 6]
```diff
@@ -62,15 +62,16 @@ the terminal automatically restarts.
 Configure the payment method
 ----------------------------
 
-From the Point of Sale application, go to :menuselection:`Configuration --> Settings --> Payment
-terminals` and activate the *Worldline* payment terminal.
+First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst:65]:
```suggestion
First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```
Same

File: content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst [hunk 7]
```diff
@@ -62,15 +62,16 @@ the terminal automatically restarts.
...
+payment method <../../payment_methods>`. Set the journal as :guilabel:`Bank` and select
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst:66]:
```suggestion
payment method <../../payment_methods>`. Set the journal type as :guilabel:`Bank` and select
```

File: content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst [hunk 8]
```diff
@@ -62,15 +62,16 @@ the terminal automatically restarts.
...
+device in the :guilabel:`Payment Terminal Device` field.
 
 .. image:: worldline/worldline-payment-terminals.png
-   :align: center
 
-Then, go to :menuselection:`Configuration --> Payment methods` and create a new payment method for
-*Worldline*. Select the payment terminal *Worldline* and your payment terminal device on your
-*Payment Method form*.
+Once the payment method is created, you can select it in your POS settings. To do so, go to the
+:ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment method
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst:73]:
```suggestion
:ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment method
```
Why would you :ref: the "Configuration --> Settings" menu when it's at the top of the page and always accessible ? It takes longer for the customer to click on the ref and read the page than just read a menuselection of 3 levels (PoS --> Configuration --> Settings)

**Comment by toaa-odoo** [comment]
@LoredanaLrpz As discussed, I'll let you read through the PR, make changes, and ping me again when you you're done :)

**Comment by LoredanaLrpz** [comment]
Hi @toaa-odoo, thanks for the review; the PR is ready for another look. Just a quick request: in the future, could we coordinate before making changes to my branches? I prefer a collaborative approach. That aside, I've updated the content and path as discussed for better clarity.

**Comment by toaa-odoo** [comment]
LGTM, I pushed the commit because one character was beyond the 100th limit

**Review by toaa-odoo — APPROVED** [review]

**Review by xpl-odoo — APPROVED** [review]
@robodoo delegate+

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Pull request closed without merging**